image_processing/src/ui_utils.py

```diff
@@ -0,0 +1,206 @@
+from h2o_wave import Q, app, ui, main
+import os
+import ip_utils as ip
+import views.layout_utils as layouts
+import cv2
+
+
+async def do_translation(q: Q):
+    image_name = ip.plot_image(ip.image_translation(q.client.image, q.args.translate_X, q.args.translate_Y))
+    temp, = await q.site.upload([image_name])
+    os.remove(image_name)
+    q.page['transformed_image'].content = f'''![plot]({temp})\n
+            Last Translation X = {q.args.translate_X}  Y = {q.args.translate_Y}
+            '''
+
+
+async def reset_translation(q: Q):
```

Comment:
When you do resets, is there a reason you cannot just use the Original Image which is already uploaded on the server?